sys-fs/loop-aes/loop-aes-3.7p.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit linux-mod
+
+MY_P="${PN/aes/AES}-v${PV}"
+
+DESCRIPTION="Linux kernel module to encrypt disk partitions with AES cipher"
+HOMEPAGE="http://loop-aes.sourceforge.net/loop-AES.README"
+SRC_URI="mirror://sourceforge/${PN}/${MY_P}.tar.bz2"
```

Comment:
You should use their download address, the "mirror" in here doesn't work anymore like it should.